### DISCUSSION TITLE
Animated text

### DISCUSSION THREAD

**Discussion by Fefedu973** [maximmax42/Discord-CustomRP#162] [Ideas]
Add an option to change text every x seconds and display anoyther text

**Comment by maximmax42** [comment]
Maybe.

**Comment by Fefedu973** [comment]
it would be super cool ! with this we could do "multi rp" if we can change eveything every 15 seconds for example we can make 1 rp for personal (website + youtube channel etc.) one (fake) for currently listening music (like spotify) and a last one for visual studio code for example. it would be simpler than setting up a real "multi rp" support (I think it would be possible to make software that would detect all discord rp and where we could choose who is displayed to make a loop etc) but this would be a great solution until such software comes out

**Comment by Fefedu973** [comment]
and we could add an option to change the name of the discord bot via an api which would make the immersion even greater

**Reply by maximmax42** [reply]
What do you mean?

**Reply by Fefedu973** [reply]
> What do you mean?

you know the title is the bot name but if we want to change text we want to change the discord app name too and we can do that with discord api

**Reply by maximmax42** [reply]
No, I think you can only do that from the developer portal, and the change most likely won't be immediate.

**Reply by Fefedu973** [reply]
> No, I think you can only do that from the developer portal, and the change most likely won't be immediate.

ok . then I don't know the level of complexity but if it's not possible that means that it would be necessary to make several bots with different IDs

**Reply by maximmax42** [reply]
> several bots with different IDs

In that case, the update rate will be even slower, I think 3 times in a minute, maybe less, maybe more.

**Reply by Fefedu973** [reply]
ok sad but not serious just enter a generic name for the bot that would work for everything. last request from me I think it would be possible to make a plugin for vs code that will automatically enter the right info in one of the custom rp iterations (software using the rich presence of discord could add an api that would modify what it would take directly in custom rp)

**Reply by maximmax42** [reply]
Pretty sure there are already Rich Presence plugins for VSCode.

**Reply by Fefedu973** [reply]
yes i know but you can create an api for your software. like that the rp developers of other software can add compatibility in custom rp which would allow them to alternate between them finally I don't know if it's feasible or if it's complicated but with that I think custom rp would be the ultimate rich presence management software

**Reply by maximmax42** [reply]
Oof. Not sure I can do that.
